bun/starter/src/main.ts

```diff
@@ -1,29 +1,31 @@
-import { Client } from "node-appwrite";
+import { Client, Users } from "node-appwrite";
 
-// This is your Appwrite function
-// It's executed each time we get a request
+// This Appwrite function will be executed every time your function is triggered
 export default async ({ req, res, log, error }: any) => {
-  // Why not try the Appwrite SDK?
-  //
-  // const client = new Client()
-  //    .setEndpoint('https://cloud.appwrite.io/v1')
-  //    .setProject(Bun.env["APPWRITE_FUNCTION_PROJECT_ID"])
-  //    .setKey(Bun.env["APPWRITE_API_KEY"]);
+  // You can use the Appwrite SDK to interact with other services
+  // For this example, we're using the Auth service
+  const client = new Client()
+    .setEndpoint(Bun.env["APPWRITE_FUNCTION_API_ENDPOINT"])
+    .setProject(Bun.env["APPWRITE_FUNCTION_PROJECT_ID"])
+    .setKey(req.headers['x-appwrite-key'] ?? '');
```

Comment:
The user's API is supposed to work with a JWT or an API key right? 
What does `x-appwrite-key` contain? 

I believe we might need slightly different starters for server vs client SDKs?

Reply:
x-appwrite-key is dynamic API key. This is starter for server